ils_middleware/tasks/folio/map.py

```diff
@@ -14,17 +14,9 @@
         "uri": "work",
         "class": "bf:Person",
     },
-    "dateOfPublication": {
```

Comment:
Is this something that FOLIO doesn't support?  Is this mapping complete or do we think there is more work here?

Reply:
The `dateOfPublication` has been absorbed into the `publication` and is now in the FOLIO expected format for a publication.

Reply:
👍